e2e/mobile/page/trade/receive.page.ts

```diff
@@ -80,13 +80,6 @@ export default class ReceivePage {
     }
   }
 
-  @Step("Select account in list")
-  async selectAccount(account: string): Promise<void> {
-    const id = this.accountId(account);
```

Comment:
Should also remove `accountId = (t: string) => `test-id-account-${t}`; `that is no longer used